scripts/eurostat/health_determinants/common/euro_stat.py

```diff
@@ -0,0 +1,328 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python module is generalized to work with different Eurostat import such as
```

Comment:
Please add a README describing steps to add a support a new Eurostat data set.

It may be easier for users if all required changes could be consolidated into a few files, say, sv_config.py